logical/system_view.go

```diff
@@ -37,6 +39,17 @@ type SystemView interface {
 
 	// ReplicationState indicates the state of cluster replication
 	ReplicationState() consts.ReplicationState
+
+	// ResponseWrapData wraps the given data in a cubbyhole and returns the
+	// token used to unwrap.
+	ResponseWrapData(data map[string]interface{}, ttl time.Duration, jwt bool) (string, error)
```

Comment:
So long as the interface is internal only for now it's not a problem to do it later!

That said I wonder if, since we already know this will change at some point, this would be better as e.g. `*WrapInput` so that we don't have to change function signatures later and the calls are more clear?